sdk/iot/core/tests/cmocka/az_iot_telemetry_tests.c

```diff
@@ -0,0 +1,301 @@
+#include <az_iot_hub_client.h>
+#include <az_span.h>
+
+#include <stdio.h>
+
+#include <setjmp.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include <cmocka.h>
+
+#define TEST_CORRECT_TOPIC_PREFIX "devices/"
+#define TEST_CORRECT_TOPIC_SUFFIX "/messages/events/"
+#define TEST_CORRECT_TOPIC_MODULES_MID "/modules/"
+
+#define TEST_FQDN "myiothub.azure-devices.net"
+#define TEST_DEVICE_ID "my_device"
+#define TEST_MODULE_ID "my_module_id"
+#define TEST_USER_AGENT "api-version=2017-11-08-preview&DeviceClientType=iothubclient"
+#define TEST_PARAMS "key=value&key_two=value2"
```

Comment:
suggestion : const char*

Reply:
Using #define's allows me to compose strings and put this is literal structs. Usually I would do `const char*`